### DISCUSSION TITLE
Paste password in Windows 11 Login Screen

### DISCUSSION THREAD

**Discussion by jkirk** [rustdesk/rustdesk#5132] [General]
I use KeepassXC Password Manager for all my passwords.

I connect from Linux (Debian/bullseye) to Windows 11 (in a VM) and get a Windows Login Prompt to enter my password.
But it is not possible to paste the password.

AFAIK [Windows prevents clipboard access into secure input box and from from the Winlogon desktop](https://learn.microsoft.com/en-us/troubleshoot/windows-client/windows-security/file-system-error-when-pasting-password).

I also tried to use AutoKey to "type" the password (from the clipboard), but this does not seem to work with Rustdesk.

How to do you guys overcome this problem? Is there a workaround?

I currently use rustdesk 1.1.9 on both ends.

**Comment by 0-don** [comment]
yes I am having the exact same problem I am using my own clipboard manager which types out the passwords on a windows login but it doesnt work on rustdesk even though it works on anydesk or teamviewer
